src/components/SchedulePage/Event.js

```diff
@@ -36,7 +31,7 @@ function Event({ name, location, description }) {
 				</Typography>
```

Comment:
Two lines up, let's change the location to not have `variant='body1'`, since I think it should be using Poppins.

Likewise, we need to remove the `variant='body1'` attribute from line 134 of the `FAQSection`, since each of the questions should be in Poppins.

Reply:
My recommendation is to use `variant='body2'` for the location. It's still a kind of body, except it uses Poppins.

Be mindful, though, that when you remove `variant='body1'` or replace it with `body2`, the font size gets smaller. For location, this is probably fine; but for the FAQ headings, I believe the font size should stay at its current size.